### PR TITLE
(fix) O3-1883: Converted phone and email fields into configuration defined fields

### PR DESCRIPTION
**Pull request by vasharma05**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [x] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary
This PR introduces a fix to the ticket O3-1883. The issue found is that the UUID for the `email` person attribute was set up wrong in the code. Since the UUID was hard coded for both the `email` and `phone` fields, I removed the fields from the code and instead added them as configuration defined fields.

## Screenshots

image

## Related Issue

https://issues.openmrs.org/browse/O3-1883

## Other

*None.*

### CODE REVIEW

File: packages/esm-patient-registration-app/src/config-schema.ts [hunk 1]
```diff
@@ -177,7 +177,24 @@ export const esmPatientRegistrationSchema = {
         _description: 'For coded questions only. Provide ability to add custom concept answers.',
       },
     },
-    _default: [],
+    _default: [
+      {
+        id: 'phone',
+        type: 'person attribute',
+        uuid: '14d4f066-15f5-102d-96e4-000c29c2a5d7',
+        validation: {
```
Comment (ibacher) [inline packages/esm-patient-registration-app/src/config-schema.ts:185]: Is this validation needed?

**Review by ibacher — APPROVED** [review]
This basically LGTM.

File: packages/esm-patient-registration-app/src/config-schema.ts (hunk 1 above)
Comment (vasharma05) [reply packages/esm-patient-registration-app/src/config-schema.ts:185]: No actually, the code broke as it wasn't passed and then I fixed the code line, but forgot to remove it from here.
Thanks for the catch!

**Comment by vasharma05** [comment]
Hi @jayasanka-sack @jnsereko @anjula-sack !

The E2E test is being cancelled after 20 mins. As per my last conversation with @jnsereko on day before yesterday's coffee break, there might be a reason that it must be looking for something and hence ran in a loop. Since I've done some UI changes related to the Phone & Email, I guess this could be a reason for failure.

Can someone help me out fix this issue please?
Thanks!

**Comment by anjula-sack** [comment]
> Hi @jayasanka-sack @jnsereko @anjula-sack !
> 
> The E2E test is being cancelled after 20 mins. As per my last conversation with @jnsereko on day before yesterday's coffee break, there might be a reason that it must be looking for something and hence ran in a loop. Since I've done some UI changes related to the Phone & Email, I guess this could be a reason for failure.
> 
> Can someone help me out fix this issue please? Thanks!

Hi @vasharma05, let me take a look at this and get back to you.

**Comment by jnsereko** [comment]
> Hi @jayasanka-sack @jnsereko @anjula-sack !
> 
> The E2E test is being cancelled after 20 mins. As per my last conversation with @jnsereko on day before yesterday's coffee break, there might be a reason that it must be looking for something and hence ran in a loop. Since I've done some UI changes related to the Phone & Email, I guess this could be a reason for failure.
> 
> Can someone help me out fix this issue please? Thanks!

This is ideally because the registration page takes a lot of time to render all the required resources.

**Pull request merged**